api/identityserver.proto

```diff
@@ -125,6 +125,19 @@ message IsConfiguration {
   reserved 10; reserved "application_limits";
   reserved 11; reserved "organization_limits";
   reserved 12; reserved "user_limits";
+
+  // Restrictions contains constraints that are applied to the IS's business logic.
+  message Restrictions {
+    // Entity contains restrictions that are applied to entities.
+    message Entity {
+      // restrict_contacts forbids standard users from setting others as the administrative or technical contacts.
+      google.protobuf.BoolValue restrict_contacts = 1;
```

Comment:
I would make this more explicit. The comment is clear but the field name is not really. Maybe `allow_set_others_as_contact` ?

Also nit; no need for Go style comments here, 
```suggestion
      // Allow anyone to set others as the administrative or technical contacts.
      google.protobuf.BoolValue allow_set_others_as_contact = 1;
```

Reply:
Indeed the name could be better but I do disagree with the name suggestion slightly. Since this is within the restriction config, it makes more sense in my eyes to have variable which are capable of restriction rather than allowing it all operations.

TLDR: probably more appropriate to have something like `disallow_set_other_as_contact`

Will do the changes in a bit, regarding the `nit`, I know its rather insignificant but its mostly something for when someone wants to check the GoDoc of the generated code